package.json

```diff
@@ -35,6 +36,7 @@
     "mochawesome": "^7.1.3"
   },
   "dependencies": {
-    "axios": "^0.27.2"
+    "axios": "^0.27.2",
+    "md5": "^2.3.0"
```

Comment:
`md5` should be a dev dependency. Run the following command to fix it:
`npm i -D md5`